protocol/avdl/stellar1/common.avdl

```diff
@@ -21,6 +21,7 @@ protocol common {
     string desc; // 'Worthwhile token' or empty
     string infoUrl; // 'https://...' or empty
     string infoUrlText; // 'Exchange' for an exchange, 'View details' for anything else
+    boolean sendDisabled; // set to true if we can't send this as destination asset when we don't own it. For example when auth is required
```

Comment:
This would make more sense as a field on `Balance`. It's about a user's relationship with an asset. Would that be easy?

Reply:
yeah I think so; will do now

Reply:
(talked on chat and decided to do it later)